hydra-node/src/Hydra/ServerOutput.hs

```diff
@@ -21,7 +22,7 @@ data ServerOutput tx
   | ReadyToFanout
   | HeadIsAborted {utxo :: UTxOType tx}
   | HeadIsFinalized {utxo :: UTxOType tx}
-  | CommandFailed
+  | CommandFailed {clientInput :: ClientInput tx}
```

Comment:
This should require updating the `api.yaml`! Why don't we see a test failure?

Reply:
That's indeed puzzling (and concerning)